dts/arm/st/f3/stm32f3.dtsi

```diff
@@ -31,6 +32,42 @@
 		compatible = "mmio-sram";
 	};
 
+
+	clocks {
+		clk_hse: clk-hse {
+			#clock-cells = <0>;
+			compatible = "st,stm32-hse-clock";
+			status = "disabled";
+		};
+
+		clk_hsi: clk-hsi {
+			#clock-cells = <0>;
+			compatible = "fixed-clock";
+			clock-frequency = <DT_FREQ_M(8)>;
+			status = "disabled";
+		};
+
+		clk_lse: clk-lse {
+			#clock-cells = <0>;
+			compatible = "fixed-clock";
+			clock-frequency = <32768>;
+			status = "disabled";
+		};
+
+		clk_lsi: clk-lsi {
+			#clock-cells = <0>;
+			compatible = "fixed-clock";
+			clock-frequency = <DT_FREQ_K(40)>;
+			status = "disabled";
+		};
+
+		pll: pll {
+			#clock-cells = <0>;
+			compatible = "st,stm32f0-pll-clock";
+			status = "disabled";
```

Comment:
a "st,stm32f3-pll-clock";  would give a better understanding even if finally it is the same as stm32f0-pll-clock.
My concern is that we miss the explanation why f3 pll comes from f0

Reply:
This was made on purpose to avoid defining redundant compatibles and bindings.
I tried to compensate by an effort of documentation in bindings description.
This is a regular device tree practice that can be seen as well on Linux, so I think Zephyr community should be able to live with it.
So I propose to have a try as is and we'll be able to adjust later if indeed this ends up being too much confusing for users.